### DISCUSSION TITLE
Warn: `blogDir` doesn't exist: "projectPath\blog".

### DISCUSSION THREAD

**Discussion by SnowGuest** [facebook/docusaurus#8880] [General] [answered]
![image](https://user-images.githubusercontent.com/59474029/231125443-2f0b7178-3cf4-40ca-b1be-6cfed4de9be6.png)

What is the error that says the directory of the blog folder cannot be found.

But I have configured the path of the blog and it can be displayed normally in start and build.
This is my blog config.

```js
 presets: [
    [
      'classic',
      /** @type {import('@docusaurus/preset-classic').Options} */
      ({
        blog: {
          path: "update",
          blogSidebarCount: "ALL",
          blogTitle: "xxx",
          routeBasePath: "update",
          showReadingTime: true,
          blogSidebarTitle: "xxx",
        },
```

update directory exists.
The path is projectPath/update.

Thank you for your answer.

**Comment by SnowGuest** [comment]
Added: docusaurus@2.0.4; node@18

**Reply by slorber** [reply]
Please use Docusaurus 2.4 and provide a repro, using docusaurus.new.

We can't help on older versions and without a repro

**Reply by SnowGuest** [reply]
I think I made a typo, the current version is 2.4.0

**Reply by slorber** [reply]
so you still need to provide a repro otherwise I can't do anything

**Reply by hb20007** [reply, marked as answer]
This might help if you are trying to get rid of this warning in docs-only mode: https://github.com/facebook/docusaurus/discussions/4738#discussioncomment-13402534
